src/mame/video/undrfire.cpp

```diff
@@ -520,63 +546,61 @@ u32 undrfire_state::screen_update_cbombers(screen_device &screen, bitmap_ind16 &
 	scclayer[1] = scclayer[0] ^ 1;
 	scclayer[2] = 2;
 
-	screen.priority().fill(0, cliprect);
-	bitmap.fill(0, cliprect);   /* wrong color? */
+	u8 tc0480scp_pri[5];
+	u8 tc0620scc_pri[2];
+	u8 sprite_pri[4];
 
```

Comment:
@cam900 
I spent days to try to find a proper fix for the last video bug in cbombers: after the track selection - track is missing animation and checkerboard pattern on bottom and right side is missing.  Aka: tc0480 bg layer above fg layer.

if tc0480 bg pri code is 0x3210, change to 0x2310 (from ...priority = m_tc0480scp->get_bg_priority(); )

I can't see any other reason why it would put the top layer behind the background.  The only thing I can think of is maybe the bg pri code from the table in tc0480scp.cpp is wrong, or maybe configured differently for cbombers(?).

Is my fix a hack?  Is there an error on the bg pri table(tc0480scp_bg_pri_lookup), or is it configurable by the game?  What do you think?

best regards,
- dink

Reply:
it's seems like tc0360pri and tc0480scp interconnection configuration related from cbombers hardware.